system/greenhouse-ccloud/templates/_helper.tpl

```diff
@@ -0,0 +1,30 @@
+{{/* Get related Thanos store endpoints */}}
+{{- define "getRelatedStoreEndpoints" -}}
+{{- $name := index . 0 -}}
+{{- $root := index . 1 -}}
+{{- $stores := list }}
+{{- range $plugin := $root.Values.ingressPlugins -}}
+{{- $thanosStore := printf "thanos-grpc.%s:443" (trimSuffix "." (trimPrefix "ingress." $plugin.recordName)) -}}
+{{- if contains $name $thanosStore -}}
+{{- $stores = append $stores $thanosStore -}}
+{{- end -}}
+{{- end -}}
+{{ range $k, $v := $stores }}
+  - {{ $v }}
+{{- end }}
+{{- end -}}
+
+{{/* Get related Thanos store endpoints */}}
+{{- define "thanosStoreEndpoints" -}}
+{{- range $cluster := .Values.thanos.globalClusters -}}
+{{- $clusterPrefix := ($cluster | trunc 3) -}}
```

Comment:
feels dangerous, what about a tag or prefix value alongside globalClusters ? or are we sure prefix will always be 3 chars ? 
with getting prefix from values we could enable different prefixes per global thanos instance too

eg. st2 in the future, with thanos global storage having targets to st1 and st2 ?

Reply:
True. I like it. https://github.com/sapcc/helm-charts/pull/8349/commits/e8a8a9daa456604688a3779656d5e8f5b78979d8